tests/test_connection.py

```diff
@@ -35,9 +36,10 @@ def test_connect_bad():
 
 @pytest.mark.slow
 @pytest.mark.xfail
+@pytest.mark.skipif(sys.platform == "win32", "connect() hangs on Windows.")
```

Comment:
Is it time to introduce a constant like `WINDOWS = sys.platform == "win32"` somewhere?

Reply:
For the moment I don't see a huge reason, it's only used twice and it's pretty self-explanatory. A new const is cognitive load. But as things go on, if we start feeling the need, sure

Reply:
This is the third occurrence...
Only reason for suggesting it is I had to go look at one of the previous uses to remind myself of which approach for determining we are on Windows we settled on. :-)
Let's see how it goes. Hopefully not too many more Windows-specific things to worry about 🤞

Reply:
A `@pytest.mark.skip_win32` is easy to implement :)